source/puppet/4.9/style_guide.markdown

```diff
@@ -412,10 +412,10 @@ file {
     * => $defaults ;
 
   '/tmp/motd' :
-   content => "message of the day" ;
+   content => 'message of the day' ;
```

Comment:
We can look at whether we should standardize "no space before `;`, but for the purposes of this PR, that doesn't need to be changed.